tests/arp/test_unknown_mac.py

```diff
@@ -176,7 +177,8 @@ def flushArpFdb(duthosts, rand_one_dut_hostname, dut_disable_arp_update):
 @pytest.fixture(autouse=True)
 def populateArp(unknownMacSetup, flushArpFdb, ptfhost, duthosts, rand_one_dut_hostname,
                 toggle_all_simulator_ports_to_rand_selected_tor_m,              # noqa: F811
-                setup_standby_ports_on_rand_unselected_tor_unconditionally):    # noqa: F811
+                setup_standby_ports_on_rand_unselected_tor_unconditionally,     #noqa: F811
+                disable_ipv6):
```

Comment:
@nimicho there is a false positive on checker:

tests/arp/test_unknown_mac.py:17:1: F401 'tests.common.fixtures.ptfhost_utils.disable_ipv6' imported but unused

You can mute this complaint by following the above 2 lines.